app/components/Login.react.js

```diff
@@ -22,33 +22,11 @@ const ONPREM = 'onprem';
 window.document.title = `${build.productName} v${version}`;
 let usernameLogged = '';
 
-// http://stackoverflow.com/questions/4068373/center-a-popup-window-on-screen
-const PopupCenter = (url, title, w, h) => {
-    // Fixes dual-screen position
-    const dualScreenLeft = 'screenLeft' in window ? window.screenLeft : screen.left;
-    const dualScreenTop = 'screenTop' in window ? window.screenTop : screen.top;
-
-    const width = window.innerWidth
-        ? window.innerWidth
-        : document.documentElement.clientWidth
-            ? document.documentElement.clientWidth
-            : screen.width;
-    const height = window.innerHeight
-        ? window.innerHeight
-        : document.documentElement.clientHeight
-            ? document.documentElement.clientHeight
-            : screen.height;
-
-    const left = ((width / 2) - (w / 2)) + dualScreenLeft;
-    const top = ((height / 2) - (h / 2)) + dualScreenTop;
-    const popupWindow = window.open(url, title, `scrollbars=yes, width=${w}, height=${h}, top=${top}, left=${left}`);
-    return popupWindow;
-};
-
 class Login extends Component {
     constructor(props) {
         super(props);
         this.state = {
+            clientId: cookie.load('db-connector-client-id'),
```

Comment:
its oauth2 client id, so we can probably name it like that. DB connector in itself doesn't have a notion of client-id.